### ISSUE TITLE
[BUG] H012 false positive with equal inside element

### ISSUE THREAD

**Issue by jayvdb** [Riverside-Healthcare/djLint#39]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Windows
 - Python Version: 3,7
 - djLint Version: 0.4.9

## Issue
The following html triggers "There should be no spaces around attribute =".
Also I guess "attribute =" should be "symbol =" or "attribute equals sign (=)"

```html
<script id="errorTemplate" type="text/x-kendo-template">
        <div class="errorNotification">
            <h3>#= title #</h3>
            <p>#= message #</p>
        </div>
    </script>
```

## How To Reproduce
See above

**Comment by jayvdb** [comment]
Another false positive that looks rather similar, but could be distinct.

```html
<p>
        {% if activity.reporting_groups|length <= 0 %}<h3>{% trans 'General' %}</h3>{% endif %}
</p>
```
